docs/website/docs/plus/features/quality/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Staging: data quality & tests"
+description: Ensure the high quality of your data workflows
+keywords: ["dlt+", "data quality", "tests"]
+---
+import DocCardList from '@theme/DocCardList';
+
+dlt+ Staging is a complete testing and validation layer for data transformations, combining a local cache with schema enforcement, debugging tools, and integration with existing data workflows.
+
+With dlt+ we want to help data engineers test and validate data pipelines before putting them into production. For this, we provide two extensive suites:
```

Comment:
Hm, is there a reason we explain dlt+ Staging here? Imo we should keep that explanation to the landing page of the docs (which I would still need to add). I would write something along the lines of:

As part of dlt+ Staging, we provide tooling for testing and validating xyz... 

And the part you mention around data transformations at the top - isn't that what the transformations docs page is for?

Reply:
It's just we don't have a page with an explanation, what is the dlt+ Staging so I can reference it here. And I don't think landing page should be the single source of definition. Since the doc is called Staging: data quality, I thought we need to explain the concept

Alternative is to create Staging core concept and reference this page here